samples/grid-pro/demo/generate-chart-from-grid/demo.js

```diff
@@ -0,0 +1,111 @@
+let chart = null; // Chart is not created initially
+
+const activeCols = new Set();
+
+Grid.grid('grid', {
+    dataTable: {
+        columns: {
+            Year: [
+                2010, 2011, 2012, 2013, 2014, 2015,
+                2016, 2017, 2018, 2019, 2020, 2021, 2022
+            ],
+            'Installation & Developers': [
+                43934, 48656, 65165, 81827, 112143, 142383,
+                171533, 165174, 155157, 161454, 154610, 168960, 171558
+            ],
+            Manufacturing: [
+                24916, 37941, 29742, 29851, 32490, 30282,
+                38121, 36885, 33726, 34243, 31050, 33099, 33473
+            ],
+            'Sales & Distribution': [
+                11744, 30000, 16005, 19771, 20185, 24377,
+                32147, 30912, 29243, 29213, 25663, 28978, 30618
+            ],
+            'Operations & Maintenance': [
+                null, null, null, null, null, null, null,
+                null, 11164, 11218, 10077, 12530, 16585
+            ],
+            Other: [
+                21908, 5548, 8105, 11248, 8989, 11816, 18274,
+                17300, 13053, 11906, 10073, 11471, 11648
+            ]
+        }
+    },
+    events: {
+        cell: {
+            click: function () {
+                const yearColumnId = 'Year';
+                // Skip if clicked on Year column
+                if (this.column.id === yearColumnId) {
```

Comment:
create a `const columnId = this.column.id` and use it everywhere. You save a lot of duplicates of the `this.column.id`